web/src/features/exchanges/ExchangeTooltip.tsx

```diff
@@ -14,11 +14,16 @@ interface ExchangeTooltipProperties {
 
 export default function ExchangeTooltip(
   properties: ExchangeTooltipProperties
-): ReactElement {
-  const { key, netFlow, co2intensity } = properties.exchangeData;
+): ReactElement | null {
+  const { key, f, ci } = properties.exchangeData;
+  if (!f || !ci) {
+    return null;
+  }
```

Comment:
This will never be triggered as the ExchangeArrow function where this is called from has already exited early. I'll do a bigger revamp, code cleanup and deduplication of these tooltips in a different PR.